concourse/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "web.fullname" . }}
+  labels:
+    app: {{ template "web.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: web
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    dns: route53
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/tls-acme: "true"
+spec:
+  tls:
+  - secretName: {{ template "web.fullname" . }}-lego-tls
```

Comment:
kube-lego creates the secret, but this is to let nginx know where to find the secret containing the certificate. At least this is how we set it up for kubesignin.